dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/VelocityScriptActionlet.java

```diff
@@ -108,6 +109,7 @@ public void executeAction(final WorkflowProcessor processor,
                             "contentlet", processor.getContentlet(),
                             "content", processor.getContentlet()));
 
+            this.stop = processor.abort();
```

Comment:
I think the abort method is not really aborting it, right, so why is it not called something like isAborted?

Reply:
you totally right, but it is legacy old stuff